README.md

```diff
@@ -10,7 +10,7 @@ Princeton Ansible Playbooks
 
 ### MacOS
 
- * `brew cask install virtualbox`
+ * `brew cask install virtualbox` (The syntax for newer versions of Homebrew is `brew install --cask virtualbox`)
  * `brew install python`
```

Comment:
@hectorcorrea  can we only keep the command from the newer versions of homebrew?
`brew install --cask virtualbox`

Reply:
Done